next.config.js

```diff
@@ -3,11 +3,13 @@ const withImages = require('next-images');
 const path = require('path');
 const DIST_DIR = '../../dist/views';
 const SVG_ASSETS_DIR = './src/client/assets/svg'
+const withSass = require('@zeit/next-sass');
+const withCSS = require("@zeit/next-css");
 
 module.exports = withPlugins(
   [
     [
-      withImages({
+      withCSS(withSass(withImages({
```

Comment:
NextJS đã hỗ trợ SCSS, SASS rồi nên mình không cần plugin này nữa đâu.
Ref: https://nextjs.org/docs/basic-features/built-in-css-support#sass-support